WalletWasabi.Fluent/Views/Wallets/Home/History/TransactionDetailsView.axaml

```diff
@@ -54,7 +54,7 @@
 
       <!-- Block hash -->
       <c:PreviewItem Icon="{StaticResource block_id}"
-                     Label="Block ID"
+                     Label="Block hash"
```

Comment:
I think we can also simply say `Block`, WDYT?

```suggestion
                     Label="Block"
```

Reply:
Block hash makes more sense to me.
If someone asks me "What Block?" I would rather answer with the block height instead of the block hash.